pipelines/import_agendas/Jenkinsfile

```diff
@@ -87,7 +87,7 @@ pipeline {
         }
         stage('Clear cache') {
             steps {
-                sh "cd ${GOBIERTO}; bin/rails runner ${GOBIERTO_ETL_UTILS}/operations/gobierto/clear-cache/run.rb"
+                sh "cd ${GOBIERTO}; bin/rails runner ${GOBIERTO_ETL_UTILS}/operations/gobierto/clear-cache/run.rb --site-domain '${GENCAT_SITE_DOMAIN}' --namespace 'GobiertoData'"
```

Comment:
Should this cache namespace GobiertoPeople?

Reply:
No, currently we are not caching GobiertoPeople data, I think this clear-cache is related with the upload-dataset operation in GobiertoData: https://github.com/PopulateTools/gobierto-etl-gencat/blob/aa66bb1aeae013992761e560717b89409675e38d/pipelines/import_agendas/Jenkinsfile#L66-L82

Reply:
In that case it shouldn't be necessary because that cache is automatically removed. 

Then we should remove the clear cache step and see if everything works (I suspect it's not necessary).

Reply:
OK, I'll change the PR removing the step and deploy in staging then

Reply:
Deployed in staging, the pipeline in Jenkins worked fine